app/models/invoice.rb

```diff
@@ -1,6 +1,7 @@
 class Invoice < ApplicationRecord
     belongs_to :merchant
+    belongs_to :customer
     has_many :transactions
     has_many :invoice_items
-    has_many :items
+    has_many :items, through: :invoice_items
```

Comment:
I'm looking forward to this learning opportunity on the concept of many-to-many relationships